src/main/java/net/coderbot/iris/gui/option/IrisVideoSettings.java

```diff
@@ -1,22 +1,31 @@
 package net.coderbot.iris.gui.option;
 
+import net.coderbot.iris.Iris;
 import net.minecraft.client.options.DoubleOption;
 import net.minecraft.text.TranslatableText;
 
+import java.io.IOException;
+
 public class IrisVideoSettings {
-	public static double shadowDistance = 32.0;
+	public static int shadowDistance = 0;
 
 	// TODO: Grey out button when the shader pack is overriding the shadow distance
 	// TODO: Save this to a file
 	// TODO: Add a Sodium video settings button too.
 	public static final DoubleOption RENDER_DISTANCE = new DoubleOption("options.iris.shadowDistance", 0.0D, 32.0D, 1.0F, (gameOptions) -> {
-		return shadowDistance;
+		return (double) shadowDistance;
 	}, (gameOptions, viewDistance) -> {
-		shadowDistance = viewDistance;
+		double outputShadowDistance = viewDistance;
+		shadowDistance = (int) outputShadowDistance;
+		try {
+			Iris.getIrisConfig().save();
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
 	}, (gameOptions, option) -> {
 		double d = option.get(gameOptions);
 
-		if (d < 0.0) {
+		if (d <= 0.0) {
```

Comment:
Allowing people to set the distance to 0.0 to disable shadows is desirable behavior.